python/python/tests/test_async_client.py

```diff
@@ -6487,24 +6487,45 @@ async def test_lolwut(self, redis_client: TGlideClient):
             result = await redis_client.lolwut(2, [10, 20], RandomNode())
             assert "Redis ver. " in node_result
 
-    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("cluster_mode", [True])
     @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
-    async def test_random_key(self, redis_client: TGlideClient):
+    async def test_cluster_client_random_key(self, redis_client: TGlideClient):
```

Comment:
```suggestion
    async def test_cluster_client_random_key(self, redis_client: GlideClusterClient):
```
And you can remove assert for client type then

Reply:
Good point, will address all of these when I push to upstream